[Chore] Update docs (#63)
serokell/tzbot: README.md

> Hey, can we meet tomorrow at 6pm?

> 

Reviewer: I think we can also remove the "This project is participating in..." notice at the top of this file.

Every user on a timezone different than the sender's will receive an
"ephemeral message" like this (a message visible only to that user):

> > "tomorrow at 6pm" in Europe/Riga will be 10:00, 12 April 2022 in America/Winnipeg

Reviewer: I guess this example is outdated :sweat_smile: 

Could we replace lines 15-22 with an actual screenshot instead? I think that would be more useful.

## Features

* [x] When a Slack message is edited, the bot sends a new ephemeral message with all times
      in the edited message and a link to it.
* [x] Time references in codeblocks are ignored.
* [x] Supports messages sent to public and private channels and shared channels
      where bot is present; replies in threads are also supported.
* [x] Users can write the bot directly and get response with translated times.

> * [x] Every message in the workspace can be translated with the bot using _Translate_
>       Slack entrypoint in the message's context menu.
> * [x] If a message contains wrongly converted or unrecognized time references, users can
>       report an issue using entrypoints;

Reviewer: ```suggestion
* [x] Every message in the workspace can be translated by selecting _Translate time refs_
      from the message's context menu `⋮`.
* [x] If a message contains wrongly converted or unrecognized time references, users can
      report an issue from a message's context menu `⋮`;
```

      collected feedback can be either recorded to a file or be sent to a configured
      Slack channel.
* [x] Detects references to:
  * invalid times (e.g., "tomorrow at 1:30", but the clocks skip from 00:59 to 02:00 on that day).
  * ambiguous times (e.g. "tomorrow at 1:30", but the clocks are set back from 01:59
    to 01:00 on that day, meaning the time "1:30" will occur twice at two different offsets).
* [x] When a time reference is not invalid/ambiguous but the date can't be inferred precisely and
      any timeshifts in sender/receiver timezone are around, the user will be warned about them.
* [x] Handles references with timezones, offsets and _some_ timezone abbreviations
  (see [timezone_abbreviations.md](docs/timezone_abbreviations.md) for a full list).
    > tuesday at 11am Australia/Brisbane
    > tuesday at 11am UTC+3
    > tuesday at 11am CST

  If a word _looks_ like timezone abbreviation but is not known by the bot, the user is given
  a hint what abbreviations the sender probably meant.
* [ ] Multiple time mentions with shared parts are processed correctly, like:

>       "Tomorrow, let's meet between 10am and 11am".

Reviewer: Not sure I understood this example - what does "shared parts" mean in this example?
